style/main.css

```diff
@@ -194,6 +194,30 @@ form[role="dialog"][data-type="confirm"].lcdui-alert section {
   left: calc(50% - 17px);
 }
 
+.background-screen {
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  bottom: auto;
+  background-color: white;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-image: url("../img/icon-128.png");
```

Comment:
> Actually, I've noticed sometimes it's correct, sometimes it's a white screen...

That sounds like a race in FxOS. We might be able to work around it by using an *&lt;img>* element rather than the *background-image* CSS property. But we can do that in a followup.